### PR TITLE
Client regenerated with the latest output storage changes

### PR DESCRIPTION
**Pull request by manikraja**
- Client regenerated to accommodate output storage configuration details
- Provides customers with an option to configure their own output storage where the generated documents can be stored

### CODE REVIEW

File: src/Kmd.Logic.DocumentGeneration.Client/regenerate.ps1 [hunk 1]
```diff
@@ -1,4 +1,4 @@
-autorest --input-file=documentgeneration.swagger.json --output-folder=. --namespace=Kmd.Logic.DocumentGeneration.Client --csharp --override-client-name=InternalClient --add-credentials
+autorest --input-file=documentgeneration.swagger.json --output-folder=. --namespace=Kmd.Logic.DocumentGeneration.Client --csharp --legacy --override-client-name=InternalClient --add-credentials
```
Comment (satyajitmahapatra) [inline src/Kmd.Logic.DocumentGeneration.Client/regenerate.ps1:7]: Can we add a comment here that why `--legacy` is used?

Comment (manikraja) [reply src/Kmd.Logic.DocumentGeneration.Client/regenerate.ps1:7]: Resolved

**Comment by satyajitmahapatra** [comment]
Approved

**Review by avin101 — COMMENTED** [review]
Approved

**Review by thecorrado — APPROVED** [review]

**Pull request merged**